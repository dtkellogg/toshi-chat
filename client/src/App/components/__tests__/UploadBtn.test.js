import UploadBtn from '../UploadBtn'


describe('tests for UploadBtn Component', () => {
  test('matches snapshot', () => {
   const wrapper = shallow(<UploadBtn />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
})