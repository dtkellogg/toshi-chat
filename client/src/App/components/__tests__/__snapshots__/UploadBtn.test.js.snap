// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for UploadBtn Component matches snapshot 1`] = `
<div>
  <img
    src={Object {}}
  />
  <input
    onChange={[Function]}
    type="file"
  />
  <button
    onClick={[Function]}
  >
    Upload
  </button>
</div>
`;
